src/panels/config/automation/condition/ha-automation-condition-row.ts

```diff
@@ -217,6 +221,25 @@ export default class HaAutomationConditionRow extends LitElement {
             ></ha-automation-condition-editor>
           </div>
         </ha-expansion-panel>
+        <div
+          class="testing ${classMap({
+            active: this._testing,
+            pass: this._testingResult === true,
+            error: this._testingResult === false,
+          })}"
+        >
+          ${this._testingResult === undefined
+            ? this.hass.localize(
+                "ui.panel.config.automation.editor.conditions.testing"
```

Comment:
Condition testing is a callback, not even await. It should be super fast. Do we think this step is necessary? Don't think you'll ever see it.

Reply:
Oh, validation also takes time I guess.

Reply:
I implemented it because the button did...  Happy to remove it :)

Reply:
I think that we can remove it.